gluten-core/src/main/scala/io/glutenproject/expression/TernaryExpressionTransformer.scala

```diff
@@ -108,6 +108,39 @@ case class StringLocateTransformer(
   }
 }
 
+case class StringSplitTransformer(
+    substraitExprName: String,
+    srcExpr: ExpressionTransformer,
+    regexExpr: ExpressionTransformer,
+    limitExpr: ExpressionTransformer,
+    original: StringSplit)
+  extends ExpressionTransformer
+  with Logging {
+
+  override def doTransform(args: java.lang.Object): ExpressionNode = {
+    // In velox, split function just support tow args, not support limit arg for now
+    if (!BackendsApiManager.isVeloxBackend) {
```

Comment:
Currently, we prefer separating the impl. if there are some differences for CH/velox backend, i.e., create another `StringSplitTransformer` for velox backend and handle its supported/unsupported cases. See `genAliasTransformer`.
Thanks!

Reply:
Done